WikiDrips/WikiDocViewController.swift

```diff
@@ -41,8 +43,8 @@ class WikiDocViewController: UIViewController, WKUIDelegate {
             print("Error: cannot create document URL")
             return nil
         }
-        // TODO: Add caching. Apparently WKWebViews don't support URLRequest cachePolicy
-        return URLRequest(url: url, cachePolicy: .returnCacheDataElseLoad)
+        
+        return URLRequest(url: url, cachePolicy: .useProtocolCachePolicy)
```

Comment:
Since this is ignored anyway, maybe its better to use `URLRequest(url: url)` instead?